mockito-core/src/main/java/org/mockito/internal/stubbing/InvocationContainerImpl.java

```diff
@@ -83,7 +83,11 @@ public void addConsecutiveAnswer(Answer<?> answer) {
     }
 
     Object answerTo(Invocation invocation) throws Throwable {
-        return findAnswerFor(invocation).answer(invocation);
+        StubbedInvocationMatcher matcher = findAnswerFor(invocation);
+        if (matcher == null) {
+            throw new NullPointerException("No answer found for: " + invocation);
```

Comment:
Instead of throw a NPE here, let's use `MockitoException` instead and provide a more descriptive error message:

```java
throw new MockitoException("Unable to find answer for " + invocation + ". Did you configure a stubbing for this method?");
```

Reply:
Thank you for the feedback, made necessary changes.